crates/pop-cli/src/commands/clean.rs

```diff
@@ -225,8 +265,31 @@ mod tests {
 			)
 			.expect_confirm("Are you sure you want to remove the selected artifact?", false)
 			.expect_outro("ℹ️ No artifacts removed");
+		
+		CleanCacheCommand { cli: &mut cli, cache, all: false }.execute()?;
 
-		CleanCacheCommand { cli: &mut cli, cache }.execute()?;
+		cli.verify()
+	}
+
+	#[test]
+	fn clean_cache_cleans_dir_when_all_flag_specified() -> Result<()> {
+		let temp = tempfile::tempdir()?;
+		let cache = temp.path().to_path_buf();
+		let artifacts = ["polkadot-parachain", "pop-node"]
+			.map(|a| cache.join(a));
+		for artifact in &artifacts {
+			File::create(cache.join(artifact))?;
+		}
+
+		let mut cli = MockCli::new()
+			.expect_confirm(format!("Would you like to cleanup all cached artifacts...\n {:?} \n", artifacts), true)
```

Comment:
Nice test. One fixed the comment above with the imports, you need to replicate the same message as above to assert the message is the same.